src/main/java/salesforce/ui/pages/legalentity/NewLegalEntityClassicVersion.java

```diff
@@ -0,0 +1,173 @@
+package salesforce.ui.pages.legalentity;
```

Comment:
You should add copyright to match our teams conventions.

Reply:
okay